dev/data/dashboard.json

```diff
@@ -3057,5 +3057,192 @@
         }
       ]
     }
+  },
+  {
```

Comment:
I didn't add any e2e tests for this because we get pretty good coverage from jest tests and visual tests from storybook. However, I added a test dashboard for ease of debugging and playing around with legends in the app context.